### PR TITLE
[DependencyInjection] inline conditional statements

### PR DESCRIPTION
**Pull request by hhamon**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | ~
| License       | MIT
| Doc PR        | ~

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
why not 2.7?

**Comment by fabpot** [comment]
I don't see the point of such changes.

File: src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php [hunk 1]
```diff
@@ -1029,11 +1029,7 @@ private function addMethodMap()
     private function addAliases()
     {
         if (!$aliases = $this->container->getAliases()) {
-            if ($this->container->isFrozen()) {
-                return "\n        \$this->aliases = array();\n";
-            } else {
```
Comment (HeahDude) [inline src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php]: Maybe just remove the `else`?

**Comment by nicolas-grekas** [comment]
merged so that we don't spend more time on this (it will come up sometime later otherwise :) )
thanks @hhamon

**Pull request merged**